src/style.css

```diff
@@ -1 +1,71 @@
-/* Styles go here */
+* {
+  margin: 0;
```

Comment:
do not remove margin for all selectors.
only for elements where it is necessary